src/components/AssetHeader/AssetDescription.tsx

```diff
@@ -27,6 +27,11 @@ export const AssetDescription = ({ assetId }: AssetDescriptionProps) => {
   const query = useGetAssetDescriptionQuery(assetId)
   const isLoaded = !query.isLoading
 
+  // Collapse about section any time description changes
+  useLayoutEffect(() => {
```

Comment:
What's the rationale to not using `useEffect` here? In theory, `useLayoutEffect` will be ran before paint while `useEffect` is ran after the paint thread has flushed, but in effect (pun intended), I tried using `useEffect` with the exact same result.

Did you find any flow where the render thread is under heavy load and `useEffect` didn't work?